app/jb.py

```diff
@@ -184,15 +182,17 @@ def status_update_callback_wrapper(event, data):
         nl2dsl.nl2dsl()
 
         errors = nl2dsl.validate_dsl()
```

Comment:
Change the name of the variable since it carries errors, warnings, etc.

Reply:
Changed `errors` into `issues`.